feature-author/src/main/java/com/google/samples/apps/nowinandroid/feature/author/AuthorScreen.kt

```diff
@@ -157,8 +157,7 @@ private fun AuthorHeader(author: Author) {
             modifier = Modifier
                 .size(216.dp)
                 .align(Alignment.CenterHorizontally)
-                .clip(CircleShape)
-                .padding(bottom = 12.dp),
```

Comment:
I think the `.padding(bottom = 12.dp)` is intended. But since with a fixed `.size()`,  `.padding()` will just resize the contents (including any `Modifier` change) following its declaration with -ve padding offsets, placing `.padding()` before `.size()` should solve the issue.

```suggest
AsyncImage(
            modifier = Modifier
                .padding(bottom = 12.dp)
                .size(216.dp)
                .align(Alignment.CenterHorizontally)
                .clip(CircleShape),
            ..
        )
```

<table>
<tr><td>Before</td><td>After</td></tr>
<tr>
<td>
<img src="https://user-images.githubusercontent.com/5493586/168221227-4867cee1-7ea8-4123-b715-8a2cea8c1c3d.png"/>
</td>
<td>
<img src="https://user-images.githubusercontent.com/5493586/168221483-5bf2816f-c2f1-40da-9a01-c8469fc1af6a.png" />
</td>
</tr>
</table>

Reply:
I see thanks, I've applied what you said by changing the modifier placement